lib/src/common/model/repos_releases.dart

```diff
@@ -21,16 +24,24 @@ class Release {
   @JsonKey(name: "zipball_url")
   String zipballUrl;
 
+  /// The endpoint for uploading release assets.
+  /// This key is a hypermedia resource. https://developer.github.com/v3/#hypermedia
+  @JsonKey(name: "upload_url")
+  String uploadUrl;
+
   /// Release ID
   int id;
 
+  @JsonKey(name: "node_id")
+  String nodeId;
+
   /// Release Tag Name
   @JsonKey(name: "tag_name")
   String tagName;
 
   /// Target Commit
   @JsonKey(name: "target_commitish")
-  String targetCommitsh;
+  String targetCommitish;
```

Comment:
I put up this PR to address this https://github.com/DirectMyFile/github.dart/pull/169